pos_remove_pos_category/views/pos_category_view.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <record id="product_category_form_view" model="ir.ui.view">
+        <field name="model">product.category</field>
+        <field name="inherit_id" ref="product.product_category_form_view"/>
+        <!-- Priority to avoid to break account and logistics fields-->
+        <field name="priority">20</field>
+        <field name="arch" type="xml">
+            <field name="parent_id" position="after">
+                <field name="available_in_pos"/>
+            </field>
+            <xpath expr="//div[contains(@class,'oe_title')]" position="before">
+                <field name="image_medium"
+                       widget="image"
+                       class="oe_avatar"
+                       options='{"size": [100, 100]}'/>
+            </xpath>
+        </field>
+    </record>
+
+    <record id="product_category_list_view" model="ir.ui.view">
+        <field name="model">product.category</field>
+        <field name="inherit_id" ref="product.product_category_list_view"/>
+        <field name="arch" type="xml">
+            <xpath expr="//field[@name='display_name']" position="after">
```

Comment:
If the XPath tag has no any specific path then we can directly use the field name without xpath like,
`<field name="display_name" position="after">`